### DISCUSSION TITLE
CloudAMQP and other vendor-specific questions do not belong here

### DISCUSSION THREAD

**Discussion by AmosSpark** [rabbitmq/rabbitmq-server#12242] [Questions] [answered]
I am setting up an open source project and it requires me to provide erlang cookie for rabbitMQ, do I need to bother providing it since the instance is managed on cloudAMQ or I need to install RabbitMQ locally so I can provide an erlang cookie alongside other credentials?

**Comment by johanrhodin** [comment]
Please contact CloudAMQP support directly: support@cloudamqp.com

**Comment by michaelklishin** [comment, marked as answer]
@AmosSpark I find it insulting that you expect the RabbitMQ core team, who provide ≈ 97% of contributions to RabbitMQ development, to help you for free while you pay a different company who contributes next to nothing (in the grand scheme of things).
